app/controllers/users.js

```diff
@@ -36,3 +36,16 @@ exports.login = async (req, res, next) => {
     next(error);
   }
 };
+
+exports.getUsers = async (req, res, next) => {
+  try {
+    const pagination = req.query;
+    let users = await userService.getUsers(pagination);
+    if (users.length > 0) {
```

Comment:
I think that it is not necessary to rewrite the users variable due to data immutability issues instead of a let, I would use a const for serializer and create a new variable example  usersResponse , I also consider that the if is not necessary instead Could you use a ternary example users.lenght ? serializer: []; no need for condition> 0

Reply:
logic simplified